tensorflow/core/framework/kernel_def.proto

```diff
@@ -16,6 +16,12 @@ message KernelDef {
 
   // Type of device this kernel runs on.
   string device_type = 2;
+ 
+  // Sub-type of device this kernel runs on.
+  // For first-party devices, subdevice_type is the same as device type.
+  // For PluggableDevices(third-party devices), subdevice_type is registered 
```

Comment:
s/PluggableDevices(third-party devices)/PluggableDevices (third party devices)/

Reply:
done, thanks for the reminder